src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1387,6 +1387,8 @@
   "com.minecolonies.coremod.research.requirementnotmet": "Minimum research requirement not met!",
   "com.minecolonies.coremod.research.started": "Research successfully started!",
   "com.minecolonies.coremod.research.havetounlock": "Build a University and research this hut to unlock it!",
+  "com.minecolonies.coremod.research.item1": "§cTo build the %s, you first need",
+  "com.minecolonies.coremod.research.item2": "§cto complete its research in the University.",
```

Comment:
This is all looking good. My final nitpick, perhaps more of an opinion:
Seeing as the building name is right above this message in the tool-tip, it doesn't need to be included again. Also I think we can get it on one line. My suggestion is "Requires research at the University for hut construction"

Reply:
While i would like a message that is short enough that it can reasonably be one line, i also like the name of the Hut being in the tooltip.
Not sure what i will do about that yet.

Reply:
I tried shortening the tooltip until it could reasonably fit one line, but the shortest i could get it to still needs two lines to match the few vanilla tooltips.

Reply:
The name of the hut in the message is just unnecessary repetition, thus it is the least important piece of information in the tool-tip. The player will simply gloss over that information. But hey if you really want it there I suppose it makes very little difference either way.

Reply:
I tried removing it, but imo that makes it too generic to fit what it should say.
Imo it stops mainly saying "You need to research me" and instead mainly says "I am a generic tooltip most of these huts have", thats why i feel like it should stay, even if it is unnecessary information.

Reply:
I think it might work if the tooltip was even shorter, but i don't think there is anything but the hut name that can be removed without changing the message.

Reply:
Fair enough, I can see that side of things.

I mean for super bonus points you could say which research is required instead, but I don't think that's worth the effort :smile:

Reply:
That would be nice, but the `addInformation` method doesn't have access to that, and i don't think storing that in the block just for this is worth it.
Especially since the block currently can't really get that information either, because the effect doesn't seem to store that.
And i don't think a solution that requires storing that in the block anyways is worth that much effort :)